tests/kani/Vectors/push.rs

```diff
@@ -4,7 +4,7 @@
 // Push 5 elements to force the vector to resize, then check that the values were correctly copied.
 #[kani::proof]
 fn main() {
-    let mut v = Vec::new();
+    let mut v: Vec<isize> = kani::vec::any_vec::<_, 0>();
```

Comment:
I'm not sure this makes sense here, given that we're starting with an empty vec. What does it mean to be an unconstrained empty vec?

Reply:
Tracking #1329.